probe.c

```diff
@@ -61,120 +69,161 @@ void probe_init()
 
 void probe_check()
 {
-  if (probe_get_active_sensor_state())
+  if (probe_get_active_sensor_state()) {
```

Comment:
I agree, but I put the if-statement there because of the TODO below this line, which will also be part of this if-statement.